common/lib/capa/capa/responsetypes.py

```diff
@@ -833,9 +939,32 @@ def get_score(self, student_answers):
         """
         # log.debug('%s: student_answers=%s, correct_choices=%s' % (
         #   unicode(self), student_answers, self.correct_choices))
+        
+        tree = self.xml
+        problem_xml = tree.xpath('.')
+
+        # Partial credit type - can set 'points' only at the moment.
+        credit_type = problem_xml[0].get('partial_credit', default=False)
+
+        try:
+            credit_type = str(credit_type).lower().strip()
+        except ValueError:
+            _ = self.capa_system.i18n.ugettext
```

Comment:
Ugh, ok. It's a bummer of a style but ok to leave as is

On Tue, Jun 23, 2015 at 4:03 PM, Colin-Fredericks <notifications@github.com>
wrote:

> In common/lib/capa/capa/responsetypes.py
> <https://github.com/edx/edx-platform/pull/8574#discussion_r33084882>:
>
> > @@ -833,9 +939,32 @@ def get_score(self, student_answers):
> >          """
> >          # log.debug('%s: student_answers=%s, correct_choices=%s' % (
> >          #   unicode(self), student_answers, self.correct_choices))
> > +
> > +        tree = self.xml
> > +        problem_xml = tree.xpath('.')
> > +
> > +        # Partial credit type - can set 'points' only at the moment.
> > +        credit_type = problem_xml[0].get('partial_credit', default=False)
> > +
> > +        try:
> > +            credit_type = str(credit_type).lower().strip()
> > +        except ValueError:
> > +            _ = self.capa_system.i18n.ugettext
>
> Oh, the underscore. Sorry, I was confused as to which line was being
> commented on.
>
> This is another one of those pieces that I copied from elsewhere. Honestly
> I have no idea why that's there, but the same pattern appears in literally
> 25 other places in the file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8574/files#r33084882>.
>